discovery-provider/src/tasks/index_aggregate_user.py

```diff
@@ -360,16 +370,32 @@ def update_aggregate_table(
                 f"index_aggregate_user.py | most_recent_indexed_aggregate_block: {most_recent_indexed_aggregate_block}"
             )
 
+            elapsed_time = get_elapsed_time_redis(redis,
+                index_aggregate_user_last_refresh_completion_redis_key)
+
+
             with db.scoped_session() as session:
                 latest_indexed_block_num = get_latest_blocknumber(session, redis)
-                start_time = time.time()
+
                 if (
                     not most_recent_indexed_aggregate_block
-                    or latest_indexed_block_num % REFRESH_COUNTER == 0
                 ):
-                    # re-create entire table
+                    # repopulate entire table
+                    logger.info(f"index_aggregate_user.py | Repopulating {table_name}")
                     most_recent_indexed_aggregate_block = 0
                     session.execute("TRUNCATE TABLE {}".format(table_name))
+                    redis.set(
```

Comment:
we should move these redis.set calls after the session.execute. trying to avoid a condition like variable gets set in redis but the worker gets restarted before completing so the db isn't actually updated. we should move it near the other redis.set after the execute

Reply:
yea moved it